synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -74,6 +74,8 @@ def __init__(self, hs, media_repo, media_storage):
             treq_args={"browser_like_redirects": True},
             ip_whitelist=hs.config.url_preview_ip_range_whitelist,
             ip_blacklist=hs.config.url_preview_ip_range_blacklist,
+            http_proxy=os.getenv("http_proxy"),
```

Comment:
why the different capitalisation?

Reply:
just because these are the conventional names: see "ENVIRONMENT" under https://curl.haxx.se/docs/manpage.html for example.